src/GraphQL/Types/Scalars/TimeSpanMillisecondsGraphType.cs

```diff
@@ -18,21 +19,14 @@ public TimeSpanMillisecondsGraphType()
                 "The `Milliseconds` scalar type represents a period of time represented as the total number of milliseconds.";
         }
 
-        /// <inheritdoc/>
-        public override object Serialize(object value) => value switch
-        {
-            TimeSpan timeSpan => (long)timeSpan.TotalMilliseconds,
-            int i => (long)i,
-            long _ => value,
-            _ => null
-        };
-
         /// <inheritdoc/>
         public override object ParseLiteral(IValue value) => value switch
         {
             IntValue intValue => TimeSpan.FromMilliseconds(intValue.Value),
             LongValue longValue => TimeSpan.FromMilliseconds(longValue.Value),
```

Comment:
By the way `TimeSpan.FromMilliseconds(long.MaxValue)` will throw:
```
System.OverflowException: 'TimeSpan overflowed because the duration is too long.'
```

The same for `FromSeconds`.

Reply:
👍